test/SlowTests/MailingList/DocumentQueryIncludeAndStreamTest.cs

```diff
@@ -67,7 +68,7 @@ select new
             }
         }
 
-        [Fact]
+        [RavenFact(RavenTestCategory.Querying)]
```

Comment:
Please adjust the assertion in test: https://jenkins.hibernatingrhinos.com/job/PR_Tests/9034/testReport/junit/SlowTests.Tests/TestsInheritanceTests/AllTestsShouldUseRavenFactOrRavenTheoryAttributes/

Reply:
Fixed